### DISCUSSION TITLE
Hydraulic conductivity in APSIMX

### DISCUSSION THREAD

**Discussion by KeLiu7** [APSIMInitiative/ApsimX#9307] [Q&A] [answered]
There are two KS as shown below, I'm not sure if they are the same. 

Both KS have an impact on watertable depth, I am not sure if we need to adjust both or just one of them? Or should the KS value in SWIM3 be consistent with the value in Soil? @hut104 Could you please provide some suggestions? Many thanks

![image](https://github.com/user-attachments/assets/a9c54497-de9a-4664-ae29-40182e139e23)
![image](https://github.com/user-attachments/assets/e1677167-3bd3-4072-8c21-fac93ab5ff5a)

**Comment by hut104** [comment, marked as answer]
Ks is the saturated hydraulic conductivity. Kdul is the hydraulic conductivity at DUL.
For a detailed response see the SWIM paper

SWIM3: Model use, calibration, and validation
NI Huth, KL Bristow, K Verburg
Transactions of the ASABE 55 (4), 1303-1313
